### ISSUE TITLE
Star Wars: Obiwan Crash Issue in Swamps

### ISSUE THREAD

**Issue by Deadliestbridge** [xemu-project/xemu#1673]
### Title

https://xemu.app/titles/4156000f/#Star-Wars-Obi-Wan

### Bug Description

Game Crashes the Emulator when loading the Jin'Ha Swamp Level 80% consistency ignore the assertions has no affect like in previous versions  I've seen line 1904 and 1910
![image](https://github.com/xemu-project/xemu/assets/102270510/4411ea6f-cb41-4e27-9a2a-189de146d080)
![image](https://github.com/xemu-project/xemu/assets/102270510/f18e2d18-1c7c-426e-8990-a9c2b2e6a174)

### Expected Behavior

![image](https://github.com/xemu-project/xemu/assets/102270510/525216f9-fad4-460d-b866-970b2b6b735e)

### xemu Version

7.127

### System Information

OS: Windows 10
GPU: 4070
CPU: 5800x

**Comment by Deadliestbridge** [comment]
This Crash has returned as of 0.8.75 along with a new issue, it then hard crashes Xemu after selecting ignore

![Image](https://github.com/user-attachments/assets/dbdb07c8-f90c-405b-83bd-0d875c24ac5e)

![Image](https://github.com/user-attachments/assets/681f1d8e-ce0a-4240-81e2-a2fca95d6dcf)

**Comment by Darth-NOD** [comment]
Hey, @Deadliestbridge did you manage to beat the game on xemu? Or maybe on CxBx Reloaded?

**Comment by Deadliestbridge** [comment]
> Hey, [@Deadliestbridge](https://github.com/Deadliestbridge) did you manage to beat the game on xemu? Or maybe on CxBx Reloaded?

Yes it was on version 8.40 something its regressed and isn't fully completable anymore but the trade off is some visual issues which were fixed later in later releases. If you're fine texture lines and some odd shadows I'd recommend using the older version.
